src/platform/zc702_base/pl.dtsi

```diff
@@ -0,0 +1,25 @@
+/*
+ * CAUTION: This file is automatically generated by Xilinx.
+ * Version:
+ * Today is: Tue Aug 20 07:12:28 2019
+ */
+
+
+/ {
+	//#amba_pl: amba_pl {
+	//#	#address-cells = <1>;
+	//#	#size-cells = <1>;
+	//#	compatible = "simple-bus";
+	//#	ranges ;
+	//#	axi_intc_0: interrupt-controller@70000000 {
+	//#		#interrupt-cells = <2>;
+	//#		clock-names = "s_axi_aclk";
+	//#		clocks = <&clkc 15>;
+	//#		compatible = "xlnx,axi-intc-4.1", "xlnx,xps-intc-1.00.a";
+	//#		interrupt-controller ;
+	//#		reg = <0x70000000 0x10000>;
+	//#		xlnx,kind-of-intr = <0xffffffff>;
+	//#		xlnx,num-intr-inputs = <0x20>;
+	//#	};
+	//#};
```

Comment:
Hi Min,
the default pl.dtsi generated by petalinux will have the axi_intc node. The same node is present in the system-user.dtsi file. I want to overwrite the default pl.dtsi file with this empty file so that petalinux uses axi intc dts node from system-user.dtsi file